src/net/sourceforge/kolmafia/Modifiers.java

```diff
@@ -1581,7 +1627,11 @@ public boolean set(final Modifiers mods) {
           "Throne",
           "UnbreakableUmbrella");
```

Comment:
Should doubledBySquintChampagne be a `Set<ModifierType>`? Set#contains takes an Object, and it's unlikely that anything casts its argument from a ModifierType to a String.

Reply:
Fixed.